src/login/logind-button.h

```diff
@@ -5,6 +5,18 @@ typedef struct Button Button;
 
 #include "logind.h"
 
+typedef enum {
```

Comment:
when we define an enum like this we usually give its typedef and the enum the same name, i.e. `typedef foobar {…} foobar;`

or in other words, please change this line to: `typedef enum ButtonModifierMask {`)